assets/js/src/common/form/checkbox/group.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { InputHTMLAttributes, useState } from 'react';
+import Checkbox from './checkbox';
+
+type CheckboxValueType = string | string[] | number;
```

Comment:
Because all inputs are extending `DefinitelyTyped` which states, it can also have `string[]`. 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9a6908d05cccf32a073e93e3c39f1df3858b435f/types/react/index.d.ts#L1911